.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+Thanks for your interest in contributing to `opencontributionssite`! 💖
+
+> See [DEVELOPMENT.md](./DEVELOPMENT.md) for local development instructions.
+
+## Code of Conduct
+
+This project contains a [Contributor Covenant code of conduct](./CODE_OF_CONDUCT.md) all contributors are expected to follow.
+
+## Reporting Issues
+
+Please do [report an issue on the issue tracker](https://github.com/OpenContributionsProject/opencontributionssite/issues/new/choose) if there's any bugfix, documentation improvement, or general enhancement you'd like to see in the repository! Please fully fill out all required fields in the most appropriate issue form.
+
+## Sending Contributions
+
+Sending your own changes as contribution is always appreciated!
+There are two steps involved:
+
+1. [Finding an Issue](#finding-an-issue)
+2. [Sending a Pull Request](#sending-a-pull-request)
+
+### Finding an Issue
+
+With the exception of very small typos, all changes to this repository generally need to correspond to an [open issue marked as `accepting prs` on the issue tracker](https://github.com/OpenContributionsProject/opencontributionssite/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22).
+If this is your first time contributing, consider searching for [unassigned issues that also have the `good first issue` label](https://github.com/OpenContributionsProject/opencontributionssite/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22+label%3A%22good+first+issue%22+no%3Aassignee).
+If the issue you'd like to fix isn't found on the issue, see [Reporting Issues](#reporting-issues) for filing your own (please do!).
+
+### Sending a Pull Request
+
+Once you've identified an open issue accepting PRs that doesn't yet have a PR sent, you're free to send a pull request.
+Be sure to fill out the pull request template's requested information -- otherwise your PR will likely be closed.
+
+PRs are also expected to have a title that adheres to [commitlint](https://github.com/conventional-changelog/commitlint).
+Only PR titles need to be in that format, not individual commits.
+Don't worry if you get this wrong: you can always change the PR title after sending it.
+Check [previously merged PRs](https://github.com/OpenContributionsProject/opencontributionssite/pulls?q=is%3Apr+is%3Amerged+-label%3Adependencies+) for reference.
+
+Please try not to force-push commits to PRs that have already been reviewed.
+Doing so makes it harder to review the changes.
+We squash merge all commits so there's no need to try to preserve Git history within a PR branch.
```

Comment:
I want to include the take-action: https://github.com/bdougie/take-action

I'll open an issue and maybe a PR to see what you think about it? This way people can type ".take" on an issue and it will automatically be assigned to them

Reply:
Here's my PR for it: https://github.com/OpenContributionsProject/opencontributionssite/pull/7

Reply:
I love it! Yes please 😄